deployments/jmiller-hub/config/common.yaml

```diff
@@ -1,13 +1,45 @@
+# Use this  CRDS config,  maybe with a different S3 bucket, for scalable performance on AWS.
+# Optionally set CRDS_S3_RETURN_URL="1" to do direct-to-cal-memory CRDS transfers.
+# CRDS_PATH: "/home/jovyan/crds_cache"
+# CRDS_SERVER_URL: "https://jwst-serverless.stsci.edu"
+# CRDS_S3_ENABLED: "1"
+# CRDS_S3_RETURN_URI: "0"
+# CRDS_MAPPING_URI: "s3://dmd-test-crds/mappings/jwst"
+# CRDS_REFERENCE_URI: "s3://dmd-test-crds/references/jwst"
+# CRDS_CONFIG_URI: "s3://dmd-test-crds/config/jwst"
+# CRDS_USE_PICKLES: "0"
+# CRDS_DOWNLOAD_MODE: plugin
+# CRDS_DOWNLOAD_PLUGIN: "crds_s3_get ${SOURCE_URL} ${OUTPUT_PATH} ${FILE_SIZE} ${FILE_SHA1SUM}"
+
 jupyterhub:
+  prePuller:
+    continuous:
+      enabled: true
   singleuser:
+    image:
+      tag: roman-latest
+      pullPolicy: Always
+    lifecycleHooks:
+      postStart:
+        exec:
+          command: ["rsync", "-r", "/etc/default-home-contents/", "/home/jovyan"]
+          #command: ["gitpuller", "https://github.com/spacetelescope/jupyterhub-user-content", "roman", "."]
+    extraEnv:
+        CRDS_PATH:  /home/jovyan/crds_cache
+        CRDS_SERVER_URL: https://jwst-crds.stsci.edu
+    defaultUrl: "/lab"
     memory:
       guarantee: 512M
       limit: 1G
-  hub:
-    extraConfig:
-      jupyterlab: |
-        c.Spawner.cmd = ['jupyter-labhub']
-  proxy:
-    https:
-      enabled: false
-  defaultUrl: "/lab"
+  auth:
+    users:
+      - cslocum@stsci.edu
+      - jmiller@stsci.edu
+      - gough@stsci.edu
+      - jmatuskey@stsci.edu
+    admin:
+      users:
+      - cslocum@stsci.edu
+      - jmiller@stsci.edu
+      - gough@stsci.edu
+      - jmatuskey@stsci.edu
```

Comment:
We probably shouldn't have our emails in here, since this is public.  Maybe put a placeholder in the example file in docs/ ?

Reply:
I  just neutered  these in jmiller-hub.   The template file already has this.